host/app/components/card-catalog-editor.gts

```diff
@@ -0,0 +1,133 @@
+import Component from '@glimmer/component';
+import { getSearchResults } from '../resources/search';
+import type { ExportedCardRef } from '@cardstack/runtime-common';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import CardEditor from './card-editor';
+import ImportModule from './import-module';
+import { LinkTo } from '@ember/routing';
+import { service } from '@ember/service';
+import LocalRealm from '../services/local-realm';
+import type RouterService from '@ember/routing/router-service';
+import { RealmPaths } from '@cardstack/runtime-common/paths';
+import { Loader } from '@cardstack/runtime-common/loader';
+//@ts-ignore cached not available yet in definitely typed
+import { tracked, cached } from '@glimmer/tracking';
+//@ts-ignore glint does not think this is consumed-but it is consumed in the template
+import { hash } from '@ember/helper';
+
+interface Signature {
+  Args: {
+    ref: ExportedCardRef;
+  }
+}
+
+export default class CardCatalogEditor extends Component<Signature> {
+  <template>
+    <ul>
+      {{#each this.catalogEntry.instances as |entry|}}
```

Comment:
it would be weird if there was more than one catalog entry for a ref--i don't think we should each over this